migrations/migrations/20190903144152-album-purchase.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('albums_purchases', {
+      id: { type: Sequelize.INTEGER, unique: true, autoIncrement: true, allowNull: false, primaryKey: true },
+      userId: { type: Sequelize.INTEGER, allowNull: false, references: { model: 'users', key: 'id' } },
+      albumId: { type: Sequelize.INTEGER },
+      album_name: { type: Sequelize.STRING, allowNull: false }
```

Comment:
this is not the same as your model
also, as I said before you are mixing camel case and snake case
read this https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md in order to decide which to use